hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerTask.java

```diff
@@ -211,6 +225,8 @@ private void balance() {
           // one for sending command , one for running du, and one for
           // reporting back make it like this for now, a more suitable
           // value. can be set in the future if needed
+          LOG.info("ContainerBalancer will sleep for some time while waiting " +
+              "for updated usage information from Datanodes.");
           Thread.sleep(3 * nodeReportInterval);
```

Comment:
Would it make sense to log sleep duration, like the one before start?

Reply:
I've added duration to the logs.